pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,242 @@
+
+## Add operation bindings
+
+Operation bindings object contains information about the operation representation in Kafka (eg. the way to consume messages).
+
+The operation binding object provides a structured way to describe how a particular operation (publish, subscribe) should behave on a Kafka topic. The `groupid`, for example, is the Id of the consumer group, while the `cliendID` is the Id of the consumer within a consumer group.	
+
+These configurations are vital for distributed message consumption and load balancing among consumers. Learn more about other fields you can configure under [operations binding](https://github.com/asyncapi/bindings/tree/master/kafka#operation-binding-object).
+
+```yaml
+operations:
+  onUserSignedUp:
+    action: receive
+    channel:
+      $ref: '#/channels/userSignedUp'
+    bindings:
+      kafka:
+        bindingVersion: '0.4.0'
+        groupId:
+          type: string
+          enum: ['myGroupId']
+        clientId:
+          type: string
+          enum: ['myClientId']
+```
+
+## Add channel bindings
+
+Channel bindings provide protocol-specific information for a particular channel.
+
+These configurations may include information about how to connect with the Kafka topic, plus any specific settings or behaviors to be used. The Channel Binding Object is part of AsyncAPI's wider bindings architecture, which specifies how the API interacts with the messaging system — in this case, Kafka.
+
+In Kafka, you can specify a given topic's number of partitions or replicas therefore, enabling parallel processing of data or consumers. Learn more about other fields that you can configure under [channel bindings](https://github.com/asyncapi/bindings/tree/master/kafka#channel-binding-object).
+
+
+```yaml
+channels:
+  userSignedUp:
+    description: This channel contains a message per each user who signs up in our application.
+    address: user_signedup
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+    bindings:
+      kafka:
+        bindingVersion: '0.4.0'
+        partitions: 10
+        replicas: 2
+        topicConfiguration:
+          cleanup.policy: ["delete", "compact"]
+          retention.ms: 604800000
+          retention.bytes: 1000000000
+          delete.retention.ms: 86400000
+          max.message.bytes: 1048588
+```
+
+<Remember>
+If you need to follow specific naming conventions for example, Kafka topic name is different from channel name.
+</Remember>
+
+
+## Add message bindings
+
+Message bindings provide protocol-specific information for a specific message. Like the Kafka topic, it's associated with a message key (if any) and Kafka-specific properties. 
```

Comment:
```suggestion
Message bindings provide protocol-specific information for a specific message. For Kafka topics, this can include how message keys are used, and details about how serialized message data has been encoded. 
```